lsp-mode.el

```diff
@@ -7336,6 +7341,16 @@ When prefix UPDATE? is t force installation even if the server is present."
         t))
    update?))
 
+;;;###autoload
+(defun lsp-ensure-server (server-id)
```

Comment:
Is this needed? Can this be a wrapper of `with-demoted-errors` around `(lsp-install-sever nil server-id)`?

Reply:
The idea is to put that in your config and hitting remote endpoints each time you start emacs is not acceptable.